commission/models/commission_settlement.py

```diff
@@ -79,6 +80,15 @@ def _compute_can_edit(self):
     def action_cancel(self):
         self.write({"state": "cancel"})
 
+    def _message_get_suggested_recipients(self):
```

Comment:
Why not doing the agent auto-subscribed on creation instead? Similar to what happens with the customer when confirming a sales order

Reply:
yes I agree it's more in line with standard odoo behaviour